### PR TITLE
UI Changes: Standardised each section and buttons

### LINKED ISSUE #35: UI Enhancements - I
**Problems with UI**
- [ ]  The wordcloud span tags move out in the right with size of words is bigger. 
- [ ]  The summary displaying div tag is not scrollable.
- [ ]  Ask Button is misaligned.

**Solution**
- [ ] For wordcloud, one solution could be by adding margin to div with ```class = wordcloudpara```.
- [ ] In the div tag with ```id = summaryDiv```, add the scrollable property in ```index.html```.
- [ ] Add some margin to the top and move the button to a bit more right.

**Extra**
- [ ] #47

**To Contribute**
Follow the steps below after cloning the repo -
```c++
git fetch origin
git checkout 35-ui-enhancements-i
```
After making appropriate changes, make a PR.

### PR DESCRIPTION
**Pull request by gobbledy-gook**
### Summary

Problems Addressed - 
- [ ] The wordcloud span tags move out in the right with size of words is bigger.
- [x] The summary displaying div tag is not scrollable.
- [x] Ask Button is misaligned.
- [x] Loading for QnA

### Details and comments
Standardised each section namely - Summary, WordCloud, QnA in parts like sectionContainer, subHeading, and the inside elements.

Closes #35 #47

### CODE REVIEW

File: server.py [hunk 1]
```diff
@@ -22,7 +22,8 @@
 
 
 API_KEY = os.environ.get("GPTNEO", None)
```
Comment (AnsahMohammad) [inline server.py:24]: I think we're no longer using API_KEY, can you remove this line from this file and build.sh aswell >

**Review by AnsahMohammad — CHANGES_REQUESTED** [review]

File: server.py [hunk 2]
```diff
@@ -21,8 +21,10 @@
 cors = CORS(app)
 
 
-API_KEY = os.environ.get("GPTNEO", None)
-headers = {"Authorization": API_KEY}
+BART = os.environ.get("BART", None)
+ROBERTA = os.environ.get("BART", None)
+headersBart = {"Authorization": BART}
+headersRoberta = {"Authorization": ROBERTA}
 MONGO_URI = os.environ.get("mongo_uri","mongodb+srv://admin:admin@reportai.ks0reyi.mongodb.net/")
```
Comment (AnsahMohammad) [inline server.py:28]: yeah looking good. 
One doubt: 1) We aren't passing the headersRoberta anywhere right, do we need to declare one?

**Review by AnsahMohammad — CHANGES_REQUESTED** [review]
Looks good! , just fix the Pylint issue and good to go

File: server.py (hunk 2 above)
Comment (gobbledy-gook) [reply server.py:28]: It is for future purpose. We need that API token in future the way we needed it for bart. So just added as a provision.

**Review by AnsahMohammad — APPROVED** [review]
Good job 🔥

**Pull request merged**